### ISSUE TITLE
Incorrect reporting of space after bracket

### ISSUE THREAD

**Issue by SUPERCILEX** [shyiko/ktlint#91]
Ex:
```kt
stuff.groupBy { it }[templateId]
// kt-lint says I should have a space after the bracket, but I'm using array syntax
// where that doesn't make sense.
```

**Comment by shyiko** [comment]
Thanks for the ticket. Fixed in [0.10.1](https://github.com/shyiko/ktlint/releases/tag/0.10.1) :cake:.

**Comment by SUPERCILEX** [comment]
👍 thanks!
